dmriprep/interfaces/reports.py

```diff
@@ -109,24 +108,24 @@ def _generate_segment(self):
         dwi_files = self.inputs.dwi if isdefined(self.inputs.dwi) else []
         dwi_files = [s[0] if isinstance(s, list) else s for s in dwi_files]
 
-        counts = Counter(BIDS_NAME.search(series).groupdict()['task_id'][5:]
-                         for series in dwi_files)
+        # counts = Counter(BIDS_NAME.search(series).groupdict()['task_id'][5:]
+        #                  for series in dwi_files)
 
-        tasks = ''
-        if counts:
-            header = '\t\t<ul class="elem-desc">'
-            footer = '\t\t</ul>'
-            lines = ['\t\t\t<li>Task: {task_id} ({n_runs:d} run{s})</li>'.format(
-                     task_id=task_id, n_runs=n_runs, s='' if n_runs == 1 else 's')
-                     for task_id, n_runs in sorted(counts.items())]
-            tasks = '\n'.join([header] + lines + [footer])
+        # tasks = ''
```

Comment:
Please remove instead of commenting out.

Reply:
done